### DISCUSSION TITLE
Battery replacement needs adjustment in ecu?

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#1231] [Q&A]
I will do car battery replacement and wondering if dacia dokker with start/stop needs any adjustments in ecu? (read something about batt coding). 
if yes... can you please point me where can I found it in DDT or pyren ? 
thanks

**Comment by farhanseo80-droid** [comment]
Yes, for a Dacia Dokker with a start/stop system, replacing the car battery often requires coding or adjustment in the ECU to ensure the start/stop function and battery monitoring work correctly. You can usually perform this using DDT or PyRen software under the battery management or vehicle configuration section. For more detailed guidance and professional tips, you can [check it out](http://www.carbatteryreplacementsharjah.ae/) for reliable resources.
